core/admin/mailu/ui/templates/domain/list.html

```diff
@@ -46,8 +46,8 @@
     <td>{{ domain.users | count }} / {{ '∞' if domain.max_users == -1 else domain.max_users }}</td>
     <td>{{ domain.aliases | count }} / {{ '∞' if domain.max_aliases == -1 else domain.max_aliases }}</td>
     <td>{{ domain.comment or '' }}</td>
-    <td>{{ domain.created_at }}</td>
-    <td>{{ domain.updated_at or '' }}</td>
+    <td>{{ domain.created_at | format_date }}</td>
```

Comment:
Can you provide more information why you introduced this formatting? Did someone request this, or do you simply more like a more human readable date?
Can you also provide more information why you only format these two date columns, but do not apply this date formatting in the other locations in the admin interface where we use dates?